src/styles/Card.scss

```diff
@@ -1,39 +1,40 @@
+$card-text-color: #FFFF00;
+
+
 .card {
-  background-color: rgba(34,139,34,0.85);;
+  background-color: rgba(34,139,34,0.85);
   height: 200px;
-  width: 500px;
+  width: 90%;
   border: solid 7px black;
-  margin: auto;
   margin-bottom: 10px;
+  margin-top: 10px;
   border-radius: 25px;
   text-align: center;
   padding: 10px;
-  font-family: highway gotic;
+  font-family: highway gothic;
   color: white;
 }
 
-
 .distance-difficulty-container {
   display: flex;
   justify-content: space-between;
+  font-size: 20px;
 }
 
 .trail-name-title {
```

Comment:
Duplicated code between the classes of `trail-name-title`, park-name-title`, `distance-roundtrip-text`... could DRY this up

Reply:
Got it. Thanks @thatPamIAm